packages/payment-processor/overrides/jest/index.d.ts

```
@@ -0,0 +1,1779 @@
+    /**
+     * `expect.not.objectContaining(object)` matches any received object
+     * that does not recursively match the expected properties. That is, the
+     * expected object is not a subset of the received object. Therefore,
+     * it matches a received object which contains properties that are not
+     * in the expected object. It is the inverse of `expect.objectContaining`.
+     *
+     * Optionally, you can provide a type for the object via a generic.
+     * This ensures that the object contains the desired structure.
+     */
+    // eslint-disable-next-line @definitelytyped/no-unnecessary-generics
+    objectContaining<E = {}>(obj: E): any;
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)